### PR TITLE
Do not mount the public directory or the compiled assets get hidden in

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
the iso

Mounting the public directory from the host, was "hidding" the assets
files present in the public directory and so our images were broken.

This was only happening in the iso and not in our dev environments
because in production we have the assets precompiled and we don't
compile them on the fly, since we don't have a js engine installed.

Signed-off-by: Jordi Massaguer Pla <jmassaguerpla@suse.de>

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
This goes with https://github.com/kubic-project/velum-branding/pull/5

**Comment by jordimassaguerpla** [comment]
Also with https://github.com/kubic-project/velum/pull/579

File: manifests/public.yaml [hunk 1]
```diff
@@ -589,9 +589,6 @@ spec:
   - name: velum-dist-name
     hostPath:
       path: /usr/share/velum/PRODUCT
-  - name: velum-static-pages
-    hostPath:
-      path: /usr/share/velum/public
```
Comment (MaximilianMeister) [inline manifests/public.yaml]: we could keep this e.g. with `path: /usr/share/velum/public/static-pages` and then set `paths["public"] = "static-pages"` in https://github.com/kubic-project/velum/blob/master/config/application.rb

File: manifests/public.yaml [hunk 2]
```diff
@@ -315,7 +315,7 @@ spec:
     - mountPath: /srv/velum/public
```
Comment (MaximilianMeister) [inline manifests/public.yaml]: this mount would need to be dropped too

**Pull request merged**